src/meadowrun/deployment_manager.py

```diff
@@ -222,8 +236,32 @@ async def _get_git_code_paths(
         return [os.path.join(local_copy_path, path_to_source)], local_copy_path
 
 
+async def _get_zip_file_code_paths(
+    local_copies_folder: str, zip_file_url: str, code_paths: Sequence[str]
+) -> List[str]:
+    decoded_url = urllib.parse.urlparse(zip_file_url)
```

Comment:
We should probably add some local caching on the machine here? As it is, do you know what happens if the local `folder` already exists?